accounting/formatters/OsgScheddCpuRemovedFormatter.py

```diff
@@ -0,0 +1,82 @@
+from .BaseFormatter import BaseFormatter
+from datetime import datetime
+from collections import OrderedDict
+from pathlib import Path
+
+def hhmm(hours):
+    # Convert float hours to HH:MM
+    h = int(hours)
+    m = int(60 * (float(hours) - int(hours)))
+    return f"{h:02d}:{m:02d}"
+
+
+class OsgScheddCpuRemovedFormatter(BaseFormatter):
+
+
+    def get_table_title(self, table_file, report_period, start_ts, end_ts):
+        info = self.parse_table_filename(table_file)
+        # Format date(s)
+        start = datetime.fromtimestamp(start_ts)
+        if report_period in ["daily"]:
+            basename = Path(table_file).stem
+            start_date = start.strftime("%Y-%m-%d")
+            title_str = f"OSPool per {info['agg'].rstrip('s')} usage for jobs removed on <strong>{start_date}</strong>"
+        elif report_period in ["weekly", "monthly"]:
+            end = datetime.fromtimestamp(end_ts)
+            start_date = start.strftime("%Y-%m-%d")
+            end_date = end.strftime("%Y-%m-%d")
+            title_str = f"OSPool per {info['agg'].rstrip('s')} usage for jobs removed from <strong>{start_date} to {end_date}</strong>"
+        else:
+            end = datetime.fromtimestamp(end_ts)
+            start_date = start.strftime("%Y-%m-%d %H:%M:%S")
+            end_date = end.strftime("%Y-%m-%d %H:%M:%S")
+            title_str = f"OSPool per {info['agg'].rstrip('s')} usage for jobs removed from <strong>{start_date} to {end_date}</strong>"
+        return title_str
+
+    def get_subject(self, report_period, start_ts, end_ts, **kwargs):
+        # Format date(s)
+        start = datetime.fromtimestamp(start_ts)
+        if report_period in ["daily", "weekly", "monthly"]:
+            start_date = start.strftime("%Y-%m-%d")
+            subject_str = f"OSPool {report_period.capitalize()} Removed Jobs Usage Report {start_date}"
+        else:
+            end = datetime.fromtimestamp(end_ts)
+            start_date = start.strftime("%Y-%m-%d %H:%M:%S")
+            end_date = end.strftime("%Y-%m-%d %H:%M:%S")
+            subject_str = f"OSPool Removed Jobs Usage Report {start_date} to {end_date}"
+        return subject_str
+
+    def format_rows(self, header, rows, custom_fmts={}, default_text_fmt=None, default_numeric_fmt=None):
+        custom_fmts = {
+            "Min Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "25% Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "Med Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "75% Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "Max Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "Mean Hrs":   lambda x: f"<td>{hhmm(x)}</td>",
+            "Std Hrs":    lambda x: f"<td>{hhmm(x)}</td>",
+            "CPU Hours / Bad Exec Att": lambda x: f"<td>{float(x):.1f}</td>",
```

Comment:
Following changes above
```suggestion
            "CPU Hours / Exec Att": lambda x: f"<td>{float(x):.1f}</td>",
```